src/themes/BootstrapTheme/Menu/AdminMenu.php

```diff
@@ -21,7 +21,7 @@ class AdminMenu implements ContainerAwareInterface
     use ContainerAwareTrait;
     use TranslatorTrait;
 
-    public function adminMenu(FactoryInterface $factory, array $options)
+    public function menu(FactoryInterface $factory, array $options)
```

Comment:
done in https://github.com/zikula/core/pull/3293/commits/29b81e5ec4ae95e8c5f88ce2f0fd221f89353f92